crates/air-lib/interpreter-data/src/trace_pos.rs

```diff
@@ -54,3 +54,11 @@ impl TryFrom<usize> for TracePos {
         PosType::try_from(value).map(TracePos)
     }
 }
+
+impl std::ops::Sub<TracePos> for u32 {
```

Comment:
This piece of code doesn't seem to be needed.

Reply:
Moreover, it looks suspicious.  Why should we have `(u32 - TracePos): u32`?

Reply:
Good catch indeed.